waiter/src/waiter/token.clj

```diff
@@ -495,12 +508,16 @@
                         {:previous previous :status http-400-bad-request :token token :log-level :warn}))))
 
     ; Store the token
-    (let [new-token-metadata (merge {"cluster" (calculate-cluster cluster-calculator request)
+    (let [{:strs [last-update-time] :as new-token-metadata}
+          (merge {"cluster" (calculate-cluster cluster-calculator request)
                                      "last-update-time" (.getMillis ^DateTime (clock))
                                      "last-update-user" authenticated-user
                                      "owner" owner
                                      "root" (or (get existing-token-metadata "root") token-root)}
                                     new-token-metadata)
+          new-token-metadata (cond-> new-token-metadata
+                               (string? last-update-time)
+                               (assoc "last-update-time" (parse-last-update-time last-update-time)))
```

Comment:
I think using `update` rather than `assoc` here would help make it more clear that an existing value is being replaced, rather than a new entry is being added to the map.

```suggestion
                               (update"last-update-time" parse-last-update-time))
```

Reply:
Done.